google/cloud/bigquery/_helpers.py

```diff
@@ -180,6 +190,41 @@ def _int_from_json(value, field):
         return int(value)
 
 
+def _interval_from_json(
+    value: Optional[str], field
+) -> Optional[relativedelta.relativedelta]:
+    """Coerce 'value' to an interval, if set or not nullable."""
+    if not _not_null(value, field):
+        return
```

Comment:
(nit) Maybe return `None` explicitly? I almost expected the linter to complain, but it appears it didn't.